tslint.json

```
@@ -1,13 +1,14 @@
 {
   "defaultSeverity": "error",
-  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier"],
+  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier", "tslint-consistent-codestyle"],
```

Prettier must be the last item in the array. ([see](https://github.com/prettier/tslint-config-prettier))
```suggestion
  "extends": ["tslint:recommended", "tslint-react", "tslint-consistent-codestyle", "tslint-config-prettier"],
```